monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -4276,6 +4289,18 @@ object Task extends TaskInstancesLevel1 {
     def disableLocalContextPropagation: Options =
       copy(localContextPropagation = false)
 
+    /** Creates a new set of options from the source, but with
+      * the [[localContextIsolateOnRun]] value set to `true`.
+      */
+    def enableLocalContextIsolateOnRun: Options =
+      new Options(autoCancelableRunLoops, localContextPropagation, localContextIsolateOnRun = true)
```

Comment:
The other methods are using `copy`. Any reason for going against it?

Just a superficial observation, no biggie.

Reply:
Here the `copy` method only allows you to set `autoCancelableRunLoops` and `localContextPropagation` (due to bincompat we can only have copy for the original 2 parameters).